packages/namekit-react/src/components/icons/DaiSymbol.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { type SVGProps } from "react";
+
+export const DaiSymbol = (props: SVGProps<SVGSVGElement>) => {
+  return (
+    <svg
+      {...props}
+      viewBox="0 0 16 16"
+      xmlns="http://www.w3.org/2000/svg"
+      fill={props.fill ? props.fill : "inherit"}
+    >
+      <g clipPath="url(#clip0_12879_99850)">
+        <path
+          d="M8.00078 0.799805C11.9776 0.799805 15.2008 4.02364 15.2008 7.9998C15.2008 11.9766 11.9776 15.1998 8.00078 15.1998C4.02461 15.1998 0.800781 11.9763 0.800781 7.9998C0.800781 4.02364 4.02461 0.799805 8.00078 0.799805Z"
+          fill={props.fill ? props.fill : "#F5AC37"}
```

Comment:
@FrancoAguzzi another topic for discussion.

Reply:
@notrab Could you please share more details for the code review comments? Thank you :+1:

Reply:
I think we should use `fill=currentColor` instead of the hex here and just control the colour by the className on the svg component itself.

Reply:
@notrab I agree it is nice idea

Centralizes props. What if some component has multiple fill rules such as WethSymbol, in case the user wants to change the icon color, do you think about this class overwriting all the inline fill HTML props?

Reply:
@FrancoAguzzi valid point, I would vote to hardcode some values, or figure out if the SVG can be simplified for transparency, or just leave as is. My only concern in these use cases is that some icons have more customization options than others, which often leads to unwanted side effects.

Reply:
Very good point. This indeed can happen: let's take by example, Wrapped Ethereum icon, which represents a multicolor Ethereum icon... How to deal with these multi-colors? Well, nowadays we paint the whole thing as one color if user informed the color, otherwise, we leave it as native. It surely can be better set... I think it is a good opportunity to do these SVG dealing enhancements in this story: [Refine use of HTML in SVG icons](https://app.shortcut.com/ps-web3/story/25228/refine-use-of-html-in-svg-icons)

Reply:
@FrancoAguzzi I'm ok to consider this idea as resolved for this PR and for follow up separately in the story you linked above.